docs/introduction/terminology.md

```diff
@@ -90,7 +90,7 @@ For lack of a better name, it is how we currently refer to the script executed w
 #### Boot Script
 
 The script containing instructions for the VM on how to boot. The source form has the `.script` extension, and the
-"compiled" or binary form of the script has the `.boot` extension. You can get the source from from the binary form
+"compiled" or binary form of the script has the `.boot` extension. You can get the source form from the binary form
```

Comment:
Why the change from `from` to `form`?

Reply:
We could also remove the first "from/form" and it would still make sense. "You can get the source from the binary form"

Reply:
Lets phrase it like "You can get back the script source from the binary..."

Reply:
Sounds good. Gonna include that in my next round of changes.